packages/terra-form/src/Textarea.jsx

```diff
@@ -33,26 +61,44 @@ const propTypes = {
 const defaultProps = {
   defaultValue: undefined,
   name: null,
+  isAutoResizable: false,
+  isInvalid: false,
   onChange: undefined,
   required: false,
+  size: null,
```

Comment:
add default for rows

Reply:
Added here.

https://github.com/cerner/terra-core/pull/966/commits/87f3e5aa297aa27d054098780307e6cf7ba0cccb